tools/api/commands/install_profile

```diff
@@ -4,7 +4,7 @@ endpoint="v1/commands"
 jq -n \
   --arg request_type "InstallProfile" \
   --arg udid "$1" \
-  --arg payload $(cat $2|base64) \
+  --arg payload $(cat $2|openssl base64 -A) \
```

Comment:
@gerardkok What do you think about combining your change here with the other suggestion in the comments? Adding quotes surrounding the parameter. So something like this:
```
--arg payload "$(cat $2|openssl base64 -A)"
```

Reply:
Looks good to me. I suppose any solution that works on both linux and darwin is good.

Reply:
@gerardkok Can you push up an additional commit with this change? I don't believe I can push a commit to your branch otherwise I would.